include/boost/compute/random/threefry.hpp

```diff
@@ -0,0 +1,246 @@
+// Added By: Muhammad Junaid Muzammil <mjunaidmuzammil@gmail.com>
```

Comment:
Can you rename this file to `threefry_engine.hpp` in order to match the class name.